api/src/main/java/org/pmiops/workbench/workspaces/resources/WorkspaceResourceMapper.java

```diff
@@ -51,99 +50,50 @@ public interface WorkspaceResourceMapper {
 
   // a WorkspaceResource has one resource object.  Assign it and ignore all others (keep as NULL).
 
+  @Mapping(target = "cohort", source = "dbCohort")
   @Mapping(target = "cohortReview", ignore = true)
   @Mapping(target = "conceptSet", ignore = true)
   @Mapping(target = "dataSet", ignore = true)
   @Mapping(target = "notebook", ignore = true)
-  @Mapping(target = "cohort", source = "dbCohort")
-  @Mapping(target = "lastModifiedEpochMillis", source = "dbCohort.lastModifiedTime")
+  @Mapping(target = "lastModifiedEpochMillis", source = "lastModifiedTime")
   ResourceFields fromDbCohort(DbCohort dbCohort);
 
   @Mapping(target = "cohort", ignore = true)
+  @Mapping(target = "cohortReview")
   @Mapping(target = "conceptSet", ignore = true)
   @Mapping(target = "dataSet", ignore = true)
   @Mapping(target = "notebook", ignore = true)
-  @Mapping(target = "cohortReview", source = "cohortReview")
-  @Mapping(target = "lastModifiedEpochMillis", source = "cohortReview.lastModifiedTime")
+  @Mapping(target = "lastModifiedEpochMillis", source = "lastModifiedTime")
   ResourceFields fromCohortReview(CohortReview cohortReview);
 
   @Mapping(target = "cohort", ignore = true)
   @Mapping(target = "cohortReview", ignore = true)
+  @Mapping(target = "conceptSet", source = "dbConceptSet")
   @Mapping(target = "dataSet", ignore = true)
   @Mapping(target = "notebook", ignore = true)
-  @Mapping(target = "conceptSet", source = "conceptSet")
-  @Mapping(target = "lastModifiedEpochMillis", source = "conceptSet.lastModifiedTime")
-  ResourceFields fromConceptSet(ConceptSet conceptSet);
+  @Mapping(target = "lastModifiedEpochMillis", source = "lastModifiedTime")
+  ResourceFields fromDbConceptSet(DbConceptSet dbConceptSet);
```

Comment:
Is this reorder so that they are alphabetized, or is there another reason??

Reply:
If you mean the `@Mapping` fields - yes, just alphabetized